README.md

```diff
@@ -9,6 +9,56 @@ Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
 This is the Orders storage module.
 
+## PREPARATION
+
+### Tenant Initialization
```

Comment:
I suggest you rename this section to Sample Data

Reply:
Done